freqtrade/resolvers/strategy_resolver.py

```diff
@@ -38,24 +38,38 @@ def __init__(self, config: Optional[Dict] = None) -> None:
         self.strategy: IStrategy = self._load_strategy(strategy_name,
                                                        config=config,
                                                        extra_dir=config.get('strategy_path'))
+
+        # make sure experimental dict is available
+        if 'experimental' not in config:
+            config['experimental'] = {}
+
         # Set attributes
         # Check if we need to override configuration
-        attributes = ["minimal_roi",
-                      "ticker_interval",
-                      "stoploss",
-                      "trailing_stop",
-                      "trailing_stop_positive",
-                      "trailing_stop_positive_offset",
-                      "process_only_new_candles",
-                      "order_types",
-                      "order_time_in_force"
+        # (Attribute name,                           default, experimental)
+        attributes = [("minimal_roi",                   None, False),
+                      ("ticker_interval",               None, False),
+                      ("stoploss",                      None, False),
+                      ("trailing_stop",                 None, False),
+                      ("trailing_stop_positive",        None, False),
+                      ("trailing_stop_positive_offset", 0.0, False),
+                      ("process_only_new_candles",      None, False),
+                      ("order_types",                   None, False),
+                      ("order_time_in_force",           None, False),
+                      ("use_sell_signal",               False, True),
+                      ("sell_profit_only",              False, True),
+                      ("ignore_roi_if_buy_signal",      False, True),
                       ]
-        for attribute in attributes:
-            self._override_attribute_helper(config, attribute)
+        for attribute, default, experimental in attributes:
```

Comment:
nitpicking, these could be extracted to it's own methods e.g. `override_attributes` and `log_strategy_attributes`

Reply:
@shusso: Not a fan honestly :/ it is not that big and it is already encapsulated in strategy_resolver ...